change/@ni-nimble-components-18b7e8bd-7ece-49a0-a662-2a8424f6bfdd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Looks like the Angular and Blazor example apps should be updated so the selects and comboboxes have labels. Feel free to do that in this PR or a follow-up

Reply:
Updated Angular/Blazor examples with select/combobox labels.